src/generate.js

```diff
@@ -5,45 +5,44 @@ const path = require("path");
 const {forPercent, getIssuedAndExpiredDate} = require("./common");
 
 const header = [
-  { id: "email", title: "Email" },
-  { id: "firstName", title: "First Name" },
-  { id: "lastName", title: "Last Name" },
-  { id: "city", title: "City" },
-  { id: "state", title: "State" },
-  { id: "zipCode", title: "Zip Code"},
-  { id: "licenseNumber", title: "License Number" },
-  { id: "dateIssued", title: "Date Issued" },
-  { id: "expirationDate", title: "Expiration Date" },
-  { id: "suspended", title: "Suspended" },
-  { id: "revoked", title: "Revoked" },
-  { id: "revokeReason", title: "Revoke Reason" },
-  { id: "longText", title: "Long Text" }
+  { id: "email", title: "Email", generator: ({ memberless, firstName, lastName, email }) =>   memberless ? "" : forPercent(75, () => faker.internet.email(firstName, lastName, email))},
+  { id: "firstName", title: "First Name", generator: ({ firstName }) => firstName},
+  { id: "lastName", title: "Last Name", generator: ({ lastName }) => lastName},
+  { id: "city", title: "City", generator: () => faker.address.city()},
+  { id: "state", title: "State", generator: () => faker.address.stateAbbr()},
+  { id: "zipCode", title: "Zip Code", generator: () => faker.address.zipCode()},
+  { id: "licenseNumber", title: "License Number", generator: () => faker.random.uuid() },
+  { id: "dateIssued", title: "Date Issued", generator: ({ dateIssued }) => dateIssued },
+  { id: "expirationDate", title: "Expiration Date", generator: ({ expirationDate }) => expirationDate },
+  { id: "suspended", title: "Suspended", generator: () => forPercent(20, true) },
+  { id: "revoked", title: "Revoked", generator: () => false },
+  { id: "revokeReason", title: "Revoke Reason", generator: () => faker.lorem.words(10) },
+  { id: "phoneNumber", title: "Phone Number", generator: () => faker.phone.phoneNumber(forPercent(80, "###-###-####", "(###) ###-####")) },
+  { id: "longText", title: "Long Text", generator: () => faker.lorem.words(55) }
 ]
 
-exports.execute = (output, { count, email, memberless }) => {
+exports.execute = (output, { count, email, memberless, headers }) => {
```

Comment:
it is called `filter` in bin/amt-cli so either rename it there or `(output, { count, email, memberless, filter:headers })`